Components/Input/input.css

```diff
@@ -0,0 +1,41 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+.input-container {
+    color: #333;
+    font-size: var(--header5);
+    margin: var(--space-md);
+    min-width: calc(0.8 * var(--width-xxxs));
+    width: 50%;
+}
+
+.input-label {
+    display: block;
+}
+
+.input-content {
+    border: none;
+    outline: none;
+    border-bottom: 2px solid var(--dark-color);
+    margin: var(--space-xs) 0;
+    padding: var(--space-xxxs);
+    font-size: var(--paragraph);
+    width: 100%;
+    transition: 0.2s ease-in-out;
+}
+
+/* Default validator for input*/
+.default-input input:focus{
+    border-bottom : 2px solid var(--primary-color);
+}
+
+/* Error validator for input*/
+.error-input input:focus{
+    border-bottom : 2px solid var(--error-color);
+}
+
```

Comment:
Error effect color should remain all the time, not only on focus

Reply:
done 🙌🏼

> At some places you have used hyphen (-) and most of the places you've used under score (_) in the naming convention. Try to use only one of these as Pranshu has mentioned earlier about it. Other than that every thing looks good. You can merge.

thanks, bro for reminding me I have done the suggested changes